scripts/job_utils.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import absolute_import
+from builtins import int
+from future import standard_library
+standard_library.install_aliases()
+import os
+import sys
+import json
+import time
+import traceback
+import logging
+from datetime import datetime
+
+from hysds.utils import parse_iso8601
+from hysds.celery import app
+import hysds.es_util as es_util
+from elasticsearch import Elasticsearch, exceptions
+
+
+def get_timedout_query(timeout, status, source_data):
+    """Tag jobs stuck in job-started or job-offline that have timed out."""
+
+    query = {
+        "query": {
+            "bool": {
+                "must": [
+                    {
+                        "terms": {
+                            "status": status
+                        }
+                    },
+                    {
+                        "range": {
+                            "@timestamp": {
+                                "lt": "now-%ds" % timeout
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        "_source": source_data
+    }
+    return query
+
+def run_query(query):
+    
+    print(query)
+    ES = es_util.get_mozart_es()
+    result = ES.search(index="job_status-current", body=json.dumps(query))
+    print("run_query : result : \n{}".format(json.dumps(result, indent = 2)))
+    return result
+
+
+def run_query_with_scroll(query, url="localhost:9200", index = "job_status-current"):
+    client = Elasticsearch(url)
```

Comment:
instead of creating a new client, just re-use the old one like you did earlier
`ES = es_util.get_mozart_es()`